docs/user/alerting/action-types/webhook.asciidoc

```diff
@@ -72,4 +72,17 @@ Password::  An optional password. If set, HTTP basic authentication is used. Cur
 
 Webhook actions have the following properties:
 
-Body::      A json payload sent to the request URL.
+Body::      A JSON payload sent to the request URL. For example: 
++
+[source,text]
+--
+{
+  "short_description": "{{context.rule.name}}",
+  "description": "{{context.rule.description}}",
+  ...
+}
+--
+
+Note that mustache template variables are escaped so that the final JSON will be valid JSON (example: escaping double quote characters) For more information on mustache template variables, refer to
```

Comment:
```suggestion
Note that mustache template variables (the text enclosed in double braces, eg, `context.rule.name`) have their values escaped, so that the final JSON will be valid (example: escaping double quote characters). For more information on mustache template variables, refer to
```

Reply:
In addition:

- Capitalize Mustache (two places)
- Replace "eg" with "for example"
- Remove "example:"
-